cpp/include/cudf/partitioning.hpp

```diff
@@ -92,6 +95,7 @@ std::pair<std::unique_ptr<table>, std::vector<size_type>> hash_partition(
   std::vector<size_type> const& columns_to_hash,
   int num_partitions,
   hash_id hash_function               = hash_id::HASH_MURMUR3,
+  uint32_t seed                       = 0,
```

Comment:
Is zero the default seed that we really want? Is zero the same as the default seed we used previously? It might be better to define a constexpr `default_seed` value that we use here instead.

Reply:
The default seed was `0`, but I can certainly factor that out to `default_seed` (and I guess I should replace all uses of `0` with it).

Reply:
I think that would be safer.